glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -137,7 +144,11 @@ open class GleanInternalAPI internal constructor () {
      * @param enabled When true, enable metric collection.
      */
     fun setUploadEnabled(enabled: Boolean) {
-        LibGleanFFI.INSTANCE.glean_set_upload_enabled(handle, enabled.toByte())
+        if (isInitialized()) {
```

Comment:
What about `getUploadEnabled`? We should have the same non-crash guarantees there.